Emby.Server.Implementations/Dto/DtoService.cs

```diff
@@ -89,11 +90,10 @@ public BaseItemDto[] GetBaseItemDtos(IEnumerable<BaseItem> items, int itemCount,
             var channelTuples = new List<Tuple<BaseItemDto, LiveTvChannel>>();
 
             var index = 0;
-            var allCollectionFolders = _libraryManager.GetUserRootFolder().Children.OfType<Folder>().ToList();
```

Comment:
Whats the deal with removing this, I'm not necessarily opposed but it seems odd in a PR about CultureInvariant

Reply:
I second this, let's not mix different things in one PR.